policyengine_us/parameters/gov/states/nm/tax/income/deductions/standard/base_amount.yaml

```diff
@@ -0,0 +1,22 @@
+description: New Mexico provides this amount for standard deduction.
```

Comment:
```suggestion
description: New Mexico provides this amount under the standard deduction.
```

Reply:
I'd suggest `New Mexico provides this base standard deduction.`